cmd/sovereignnode/chainSimulator/sovereignChainSimulator.go

```diff
@@ -50,6 +52,8 @@ func NewSovereignChainSimulator(args ArgsSovereignChainSimulator) (chainSimulato
 		if alterConfigs != nil {
 			alterConfigs(cfg)
 		}
+
+		alteredSovConfigs = cfg.GeneralConfig.SovereignConfig
```

Comment:
You can put this line inside the `if alterConfigs != nil`

Reply:
+1

Reply:
Moved it there, but had to initialize it above as well.